lib/rubocop/cop/lint/inherit_exception.rb

```diff
@@ -0,0 +1,69 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop looks for error classes inheriting from `Exception`.
+      # It is configurable to suggest using either `RuntimeError` (default) or
+      # `StandardError instead.`
+      #
+      # @example
+      #
+      #   # bad
+      #
+      #   class C < Exception; end
+      #
+      # @example
+      #
+      #   # EnforcedStyle: runtime_error (default)
+      #
+      #   # good
+      #
+      #   class C < RuntimeError; end
+      #
+      # @example
+      #
+      #   # EnforcedStyle: standard_error
+      #
+      #   # good
+      #
+      #   class C < StandardError; end
+      class InheritException < Cop
+        include ConfigurableEnforcedStyle
+
+        MSG = 'Avoid inheriting from `Exception`. Instead use `%s`.'.freeze
```

Comment:
Use ... instead.

You can also compact this to to `Inherit from ... instead from ...`.